docs/source/adrs/0005-inline-cardid-data-into-cardid-before-pushing-to-gov-uk-paas.md

```diff
@@ -0,0 +1,62 @@
+---
+hide_in_navigation: true
+---
+
+# ADR 5: Inline cardid-data into cardid before pushing to GOV.UK PaaS
+
+Date: 2020-01-30
+
+## Status
+
+Accepted
+
+## Context
+
+The [pay-cardid](https://github.com/alphagov/pay-cardid) microservice is
+responsible for retrieving card information for a given card number.
+
+To do this, it requires a number of proprietary .csv files, which are provided
+by various payment service providers. This data is sensitive in the sense
+that it's encumbered by licence restrictions, so we need to take appropriate care
+not to make it public.
+
+The data is roughly 36 MB uncompressed, and changes rarely (a few times a year).
```

Comment:
perhaps worth noting that it is a goal to make this much more frequent (or perhaps not relevant here)